src/esp/scene/Mp3dSemanticScene.cpp

```diff
@@ -143,7 +142,9 @@ bool SemanticScene::loadMp3dHouse(
   std::string header;
   std::getline(ifs, header);
   if (header != "ASCII 1.1") {
-    LOG(ERROR) << "Unsupported House format header " << header;
+    LOG(ERROR) << "SemanticScene::loadMp3dHouse : Unsupported Mp3d House "
```

Comment:
Will this error get printed if I try to load replica or gibson semantics?  It looks like it will.  We certainly shouldn't log something as an error when all is actually fine.

Reply:
With these updated changes, this error will only trigger if this function is called directly with a filename that is assumed to reference an Mp3d file.